packages/server/routes/fields.js

```diff
@@ -310,6 +312,15 @@ const fieldFlow = (req) =>
                     "Deleting a row will also delete the file referenced by this field"
                   ),
                 },
+                {
```

Comment:
adding file filter here is clever. great idea

Reply:
Will fixed soon.